proto/tendermint/types/types.proto

```diff
@@ -227,3 +228,22 @@ message MalleatedTx {
   bytes original_tx_hash = 1;
   bytes tx               = 2;
 }
+
+// Proof represents proof of a namespace.ID in an NMT.
+// In case this proof proves the absence of a namespace.ID
+// in a tree it also contains the leaf hashes of the range
+// where that namespace would be.
+message NMTProof {
+  // start index of this proof.
+  int32 start = 1;
+  // end index of this proof.
+  int32 end = 2;
```

Comment:
Why do you need a start and end index if the proof is only for one leaf?

Reply:
that's just what nmt does, so mirrored here. NMT's current API doesn't actually allow for inclusion proofs over multiple shares, but this is purely from API perspective. The underlying code supports it. 

If we fix the API, we can fix #621